server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreRestoreIT.java

```diff
@@ -217,7 +224,7 @@ private void testRestoreFlowBothPrimaryReplicasDown(int numberOfIterations, bool
      * @param invokeFlush If true, a flush is invoked. Otherwise, a refresh is invoked.
      * @throws IOException IO Exception.
      */
-    private void testRestoreFlowMultipleIndices(int numberOfIterations, boolean invokeFlush, int shardCount) throws IOException {
+    private void testRestoreFlowMultipleIndices(int numberOfIterations, boolean invokeFlush, int shardCount) throws Exception {
```

Comment:
Unable to pull line 234 below into the comment, but similar to `testRestoreFlowBothPrimaryReplicasDown`, should we be asserting:

```java
assertEquals(shardCount * 2, getNumShards(index).totalNumShards);
```

instead of:
```java
assertEquals(shardCount, getNumShards(index).totalNumShards);
```

because the replica count on line 228 is now `1`?